src/ai/backend/manager/models/gql_models/container_registry.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import logging
+from typing import Self
+
+import graphene
+
+from ai.backend.logging import BraceStyleAdapter
+
+from ..base import BigInt
+from ..rbac import ScopeType
+from ..user import UserRole
+from .container_registry_utils import HarborQuotaManager
+from .fields import ScopeField
+
+log = BraceStyleAdapter(logging.getLogger(__spec__.name))  # type: ignore
+
+
+class CreateContainerRegistryQuota(graphene.Mutation):
+    """Added in 25.2.0."""
+
+    allowed_roles = (
+        UserRole.SUPERADMIN,
+        UserRole.ADMIN,
+    )
+
+    class Arguments:
+        scope_id = ScopeField(required=True)
+        quota = BigInt(required=True)
+
+    ok = graphene.Boolean()
+    msg = graphene.String()
+
+    @classmethod
+    async def mutate(
+        cls,
+        root,
+        info: graphene.ResolveInfo,
+        scope_id: ScopeType,
+        quota: int | float,
+    ) -> Self:
+        async with info.context.db.begin_session() as db_sess:
+            try:
+                manager = await HarborQuotaManager.new(db_sess, scope_id)
+                await manager.create(int(quota))
+                return cls(ok=True, msg="success")
+            except Exception as e:
+                return cls(ok=False, msg=str(e))
```

Comment:
It doesn’t seem like a good structure to create a new HarborQuotaManager and perform each operation every time a request comes in.
Could you explain the reasoning behind this approach?